docs/src/pages/examples/index.js

```diff
@@ -22,7 +22,7 @@ const cards = [
     links: {
       github:
         'https://github.com/hubtype/botonic-examples/tree/master/booking-platform',
-      example: '/examples/booking-platform',
```

Comment:
@manuelalferez this should work with this url, because I think that otherwise will not be indexed by Algolia. I'll leave this atm as at least is working but we will need to change it.